spec/happiness_delivered_spec.rb

```diff
@@ -0,0 +1,76 @@
+require 'happiness_delivered'
+require 'sandcream'
+
+describe HappinessDelivered do
+  let(:sandcream) { Sandcream.new }
+  
+  context 'respond to methods' do
+    it { is_expected.to respond_to(:interactive_menu) }
+    it { is_expected.to respond_to(:interface) }
+    it { is_expected.to respond_to(:print_menu) }
+    it { is_expected.to respond_to(:select_order) }
+    it { is_expected.to respond_to(:review_order) }
+    it { is_expected.to respond_to(:complete_order) }
+  # it { is_expected.to respond_to(:print_menu_options).with(1).argument }
+  end
+
+  describe '#initialize' do
+    it 'menu is a Sandcream object' do
+      expect(subject.menu).to be_a Sandcream
+    end
+
+    it 'basket is empty' do
+      expect(subject.basket).to be_empty
+    end
+
+    it 'order_total is empty' do
+      expect(subject.order_total).to be_empty
+    end
+  end
+
```

Comment:
I'm not sure if all the above tests are really necessary at this point, and that they might be too granular. If you test the behaviour of your methods properly then in theory those tests should also test everything above as part of them. Although this is something I need to work on myself so who knows 👀